src/Persistence/json/jsonconfigstore.py

```diff
@@ -0,0 +1,25 @@
+import json
+from abc import ABC
+
+from Persistence.abstractconfigstore import ConfigStore
+
+
+class JsonConfigStore(ConfigStore, ABC):
+
+    __filename = ""
+
+    def __init__(self, filename='config.json'):
+        self.__filename = filename
+        self.validate()
+
+    def validate(self):
+        with open(self.__filename, 'w+') as file:
+            file.write('{}')
+
+    def read(self):
+        with open(self.__filename, 'r') as file:
+            return json.loads(file.read())
+
+    def write(self, data: dict):
+        with open(self.__filename, 'r+') as file:
```

Comment:
Should this not have write access?
I could also use some clarification on the difference between 
r -> r+ 
w -> w+

Reply:
That was sloppy on my part.

r: read-only
r+: read and write
w: read-only, create file it it doesn't exist
w+: read and write, create if it doesn't exist
a: append
a+: append and write

Reply:
Ok, this is kind of counter-inuitive in my eyes, but looks like r+ is indeed correct :)